src/woo_publications/metadata/tests/test_waardenlijst_sync.py

```diff
@@ -0,0 +1,84 @@
+import tempfile
+from io import StringIO
+from pathlib import Path
+from uuid import uuid4
+
+from django.core.management import call_command
+from django.test import TestCase
+
+import requests
+import requests_mock
+
+from ...utils.tests.vcr import VCRMixin
+from ..models import InformationCategory
+from ..waardenlijst_sync import (
+    InformatieCategoryWaardenlijstError,
+    update_information_category,
+)
+
+
+class UpdateInformatieCategoryTestCase(VCRMixin, TestCase):
+    def test_data_gets_stored_and_turned_into_fixture(self):
+        assert not InformationCategory.objects.exists()
+
+        with tempfile.NamedTemporaryFile(suffix=".json") as file:
+            file_path = Path(file.name)
+
+            update_information_category(file_path)
+
+            with self.subTest("database populated"):
+                self.assertEqual(InformationCategory.objects.count(), 18)
+
+            with self.subTest("user content does not cause conflicts"):
+                information_category = InformationCategory.objects.order_by("pk").last()
+                information_category.identifier = "https://something.else"
```

Comment:
adding `assert information_category is not None` will fix the type checker issue :)